src/meta/api/src/schema_api_test_suite.rs

```diff
@@ -3879,6 +3883,105 @@ impl SchemaApiTestSuite {
         Ok(())
     }
 
+    #[tracing::instrument(level = "debug", skip_all)]
+    async fn test_table_lock<MT: SchemaApi>(&self, mt: &MT) -> anyhow::Result<()> {
+        let tenant = "tenant1";
+        let db_name = "db1";
+        let table_name = "tb1";
+        let table_id;
+
+        let db_table_name_ident = TableNameIdent {
+            tenant: tenant.to_string(),
+            db_name: db_name.to_string(),
+            table_name: table_name.to_string(),
+        };
+
+        let schema = || {
+            Arc::new(TableSchema::new(vec![
+                TableField::new("a", TableDataType::Number(NumberDataType::UInt64)),
+                TableField::new("b", TableDataType::Number(NumberDataType::UInt64)),
+            ]))
+        };
+
+        let options = || maplit::btreemap! {"opt-1".into() => "val-1".into()};
+
+        let table_meta = |created_on| TableMeta {
+            schema: schema(),
+            engine: "JSON".to_string(),
+            options: options(),
+            updated_on: created_on,
+            created_on,
+            ..TableMeta::default()
+        };
+
+        let created_on = Utc::now();
+
+        let req = CreateTableReq {
+            if_not_exists: true,
+            name_ident: db_table_name_ident.clone(),
+            table_meta: table_meta(created_on),
+        };
```

Comment:
use `Util` to simplify test setting up:
https://github.com/datafuselabs/databend/blob/b77f871a4c01094ade34c3db23f5ec9f11ffbedc/src/meta/api/src/schema_api_test_suite.rs#L3309-L3317